### ISSUE TITLE
Drop feedback border is different for first vs second

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#99054]
![recording (1)](https://user-images.githubusercontent.com/900690/83511096-63a1fc80-a4ce-11ea-9f9b-09d9b83bd925.gif)

It seems to me the drop border is wider in the beginning compared to the end (this is with `yarn web`).

**Comment by bpasero** [comment]
Yes, I am always on zoom 1.

**Comment by sbatten** [comment]
I get inconsistent results based on screen resolution + scaling. As I said, this is just css sizing at 2px and the browser rendering. 

Below is the reported sizing in dev tools for both separators
![image](https://user-images.githubusercontent.com/6561887/83785654-7b9c8c00-a646-11ea-951d-c4b2d1941d28.png)

**Comment by bpasero** [comment]
No, sorry. One thing which maybe happens is that it depends on the sash location between sidebar and editors? If that location happens to be on a sub-pixel coordinate, it is possible that all other calculations get weird. I think we had an idea once to never allow sub-pixel placement of sashes to avoid these issues but never pushed it.

**Comment by sbatten** [comment]
you are right, just resizing the sidebar back and forth gives you varying results
/cc @joaomoreno

**Comment by joaomoreno** [comment]
Yeah let's make the splitview always round things up.

**Comment by sbatten** [comment]
The panel header is not a split view but it is positioned inside of one since the entire workbench is made up of nested split views.

**Comment by joaomoreno** [comment]
Oh yeah... d'oh!

**Comment by joaomoreno** [comment]
This has nothing to do with the splitview, which already rounds down all layout positions. On closer inspection, the issue here lies on the panel title:

- These separators are both `::before` and `::after` pseudo elements between panel titles
- They are laid out inline between the text labels, so they get fractional positioning
- This effect is exacerbated when zooming and forces the browser to make decisions where to place them: sometimes it nails it, other times it doesn't

![image](https://user-images.githubusercontent.com/22350/87021688-cd03e200-c1d5-11ea-911c-442a0ff55aa4.png)

I highly suggest two things:

- Only use one pseudo element here, instead of two, at any given time.
- Don't use opacity for hiding/showing these pseudo elements but use `display: none`. The current solution requires all these pseudo elements to be rendered in the DOM tree at all times, which is pretty ugly considering that they will only be visible when drag and dropping.
